gtk/TreeStore.custom

```diff
@@ -475,7 +475,7 @@
 			vals [2] = new GLib.Value (iter);
 			inst_and_params.Append (vals [2]);
 			int cnt = IterNChildren (iter);
-			IntPtr new_order_ptr = Marshal.AllocHGlobal (Marshal.SizeOf (typeof (int)) * cnt);
+			IntPtr new_order_ptr = Marshal.AllocHGlobal (sizeof (int) * cnt);
```

Comment:
for all of these changes,

```
Marshal.SizeOf returns the size after the type has been marshaled, whereas sizeof returns the size as it has been allocated by the common language runtime, including any padding
```
https://msdn.microsoft.com/en-us/library/eahchzkf.aspx

Might need to undo this on the grounds of correctness.